data/campaigns/Heir_To_The_Throne/scenarios/15_The_Lost_General.cfg

```diff
@@ -471,6 +471,9 @@
             speaker=unit
             message= _ "We are desperately trying to rid these tunnels of orcs and trolls! Please help us in our quest."
         [/message]
+#ifdef EASY
+        {NARRATOR_MESSAGE (_ "The dwarves are your allies in this scenario. If you help them survive the battle, they may provide you with useful information.")}
```

Comment:
"if you help their leader survive the battle" would be clearer about exactly what the player has to do.

The reward is troops rather than information, although he gives some information too.

Reply:
How about `If you help their leader survive the battle, he may provide you with useful information`? This emphasizes the leader, but Konrad doesn't know in advance what the reward will be.